Forcing.pyx

```diff
@@ -9,30 +9,34 @@ cimport Grid
 cimport ReferenceState
```

Comment:
There are many changes in the forcings files. Is there anyone who did or could take a closer look on them?

Reply:
There are three main reasons for so many changes in forcing:
1. Changes to stretched coordinates, its changing from 'zl_half' to 'zpl_half' 
2. Adding cases (TRMM, ARM_SGP, SaturatedBubble, GATE_III and others) 
3. Adding TimeStepping class to function-calls (which need to be added to all so that the signature match) 
4. removing some unused functions relating to 'thetali' as a prognostic variable